.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,65 @@
+name: Build & Test
+
+# Run when something is pushed to main or when there's action on a pull request.
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+# Cancel running workflow if a pull request is modified. Note that head_ref is
+# undefined for pushes to main. Use run_id as fallback. This is unique for each
+# run, so runs for pushes to main are never cancelled.
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  Build-And-Test-Linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Initialize submodules
+        run: git submodule update --init --recursive
+
+      - name: Build (fastbuild)
+        run: |
+          bazel build --config=linux -- //... -//third_party/...
+
+      # Skip file_finder_test: The test works when file_finder_test is run
+      # directly, but not through bazel test. The reason is, bazel test
+      # creates symlinks of test files, but the finder ignores symlinks.
+      # Also run with --jobs 1 since some tests write to a common tmp dir.
+      - name: Test (fastbuild)
+        run: |
+          bazel test --config=linux --test_output=errors --jobs 1 //... -//third_party/... -//cdc_rsync_server:file_finder_test
```

Comment:
I think if you use the `-//foo/bar` set operation, you need to declare the end-of-args with `--` before that.

Reply:
God dammit, thanks for catching!